pkg/dns/azure/dns_test.go

```diff
@@ -1,12 +1,15 @@
 package azure_test
 
 import (
+	"reflect"
 	"testing"
 
 	"github.com/pkg/errors"
 
 	configv1 "github.com/openshift/api/config/v1"
 
+	"github.com/Azure/go-autorest/autorest/to"
```

Comment:
nit: The github.com imports are alphabetized in your other files, may want to do that here.

Reply:
comment incorporated.